static/app/views/insights/common/views/spanSummaryPage/sampleList/index.tsx

```diff
@@ -129,6 +132,33 @@ export function SampleList({groupId, moduleName, transactionRoute, referrer}: Pr
     ];
   }
 
+  const handleClickSample = useCallback(
+    (span: SpanSample) => {
+      router.push(
+        generateLinkToEventInTraceView({
+          eventId: span['transaction.id'],
+          projectSlug: span.project,
+          spanId: span.span_id,
+          location,
+          organization,
+          traceSlug: span.trace,
+          timestamp: span.timestamp,
+        })
+      );
+    },
```

Comment:
I think `navigate` is preferred here as `useRouter` is deprecated (although we can address this seperately imo)

Reply:
Agreed, but yes, one thing at a time!